codes/classical/q-ary_digits/maximum_rank_distance.yml

```diff
@@ -0,0 +1,29 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: maximum_rank_distance
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Maximum rank distance code'
+introduced: '\cite{manual_citation_minilatex:{E. M. Gabidulin, "Theory of Codes with Maximum Rank Distance", Problemy Peredachi Informacii, Volume 21, Issue 1, \emph{3–16} (1985)}}'
+
+description: 'Linear error-correcting codes \([n,k,n-k+1]_q\) over rank metric'
```

Comment:
This is a child of rank, so you can just say "Linear rank-metric code with maximum rank." However, feel free to add anything that you think makes it more convenient for people not to have to go to the rank-metric code page.

Reply:
updated